.infra/workers.ts

```diff
@@ -130,4 +130,61 @@ export const workers: Worker[] = [
     subscription: 'api-cdc',
     args: { enableMessageOrdering: true },
   },
+  {
+    topic: 'api.v1.new-comment-mention',
+    subscription: 'comment-mention-mail',
+  },
+  // Notifications
+  {
+    topic: 'community-link-rejected',
```

Comment:
Would you not want to retro fix these namings?

Reply:
These topics exist already so changing their names is out of scope